### DISCUSSION TITLE
Timelapse reworked

### DISCUSSION THREAD

**Discussion by yorkpeter** [gopro/labs#716] [General]
I would like to make a timelapse with photos from 2 hours before sunrise to 2 hours after sunrise.
I would add the time of shooting to the photos (with the BatchFoto utility) and with the Startrails utility I would create a complete MP4 video.
However, there is a problem: if I set the shutter speed to a few seconds when the sun rises the photos are burned. But if I set the shutter speed to "auto" the stars are almost invisible.
How to solve?
I specify that I could use (but I'm not like) the GoPro Lab and/or Open Gopro.

**Comment by dnewman-gpsw** [comment]
It is hard.  The auto exposure doesn't work in the deep sky 20-30s exposure modes, so at best you auto exposure is 10s at 800.  Manual exposing to 30s at 1600 ISO (3.2 stops more sensitive) or 20s at 3200 ISO (4 stops sensitive).  You could run a Labs script to switch between auto and fixed long exposure mode, but there is still the complication of when to switch. This will take experimentation. Also blending manual shutter with auto into a seamless timelapse will be hard.  The easiest solution is to use two cameras, dissolve/blend in post.

**Reply by yorkpeter** [reply]
Thanks for your answer. 
The transition from "long exposure" to "automatic exposure" is in any case abrupt and does not lead to good results.
I don't own 2 GoPro.
I think I'll create 2 different timelapses on 2 consecutive days and try to reprocess them in post-production.
But this will happen when I have the certainty of 2 consecutive days of good weather and above all not in this cold winter. In spring ...
